src/RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -156,6 +159,12 @@ public void OnPersist(StoreView snapshot, IReadOnlyList<Blockchain.ApplicationEx
                     HandleNotification(snapshot, transaction, notifyEventArgs.ScriptHash, stateItems,
                         nep5BalancesChanged, ref transferIndex);
                 }
+
+                // Handle SystemFee and NetworkFee GAS burn for each transaction
+                if (!(appExecuted.Transaction is Transaction tx)) continue;
```

Comment:
What does this mean?

Reply:
To deal with the fees for those non-null transactions.

Reply:
`appExecuted.Transaction is null`?

Reply:
![image](https://user-images.githubusercontent.com/12050350/73812536-1495d480-4818-11ea-92ad-ebb6f0eafea9.png)

For each block persisted, here the ScriptContainer is null, so the ApplicationExecuted.Transaciton is null.

Reply:
I mean, you should modify it to `appExecuted.Transaction is null`.